tensorflow/compiler/xla/service/cpu/runtime_fft_impl.h

```diff
@@ -179,7 +172,6 @@ template <int FFTRank, typename EigenDevice>
 void EigenFftWithRank(const EigenDevice& device, void* out, void* operand,
                       int32 fft_type, int64 input_batch, int64 fft_length0,
                       int64 fft_length1, int64 fft_length2) {
-  CHECK(::xla::FftType_IsValid(fft_type)) << fft_type;
```

Comment:
How about

```
if (!::xla::FftType_IsValid(fft_type)) {
  abort();
}
```

etc.?

Reply:
I added the aborts in `default` at the bottom (for both rank and type). If you think it'd be cleaner to have it at the top too, let me know.